static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -517,6 +551,29 @@ CommentBox.prototype.formatComment = function(formatStyle) {
     }
 };
 
+/**
+ * The username is used in three places:
+ *   1. header menu - read from GU_U cookie
+ *   2. discussion header - read from DAPI
+ *   3. comment - read from DAPI
+ *
+ * When the user updates the username the changes are visible only when the page is refreshed. This method
+ * temporarily updates HTML client-side so that user has instant feedback before the page is refreshed.
+*/
+CommentBox.prototype.refreshUsernameHtml = function() {
+    IdentityApi.reset();
+    var displayName = this.getUserData().displayName;
+    var userId = this.getUserData().id;
+
+    var menuHeaderUsername = document.querySelector(".js-profile-info");
+    var discussionHeaderUsername = document.querySelector("._author_tywwu_16");
+    var firstCommentUsername = document.querySelector('[data-comment-author-id=' + '"' + userId + '"]');
```

Comment:
- Single quotes on lines 568 and 569
- Qwery (`$`) could be used rather than `document.querySelector` (and it is already being included in the AMD):

```
var menuHeaderUsername = $('.js-profile-info')[0];
var discussionHeaderUsername = $('._author_tywwu_16')[0];
var firstCommentUsername = $('[data-comment-author-id=' + '"' + userId + '"]')[0];
```
...Additionally, my `discussionHeaderUsername` is updating, but not the `firstCommentUsername`This seems to be because `firstCommentUsername = Null`. I assume this is just an issue when running locally?